app/controllers/lists_controller.rb

```diff
@@ -0,0 +1,59 @@
+class ListsController < ApplicationController
+  before_action :authenticate_user!
+
+  def show
+    @lists = current_user.lists
+  end
+
+  def new
+    @list = List.new
+  end
+
+  def edit
+    @list = List.find(params[:id])
```

Comment:
With a little URL tampering, a user can edit another user's lists currently.

Write a feature spec that tests:
- a logged in attacker cannot view and edit another users list

Write a controller spec that tests:
- a logged in attacker cannot delete another users list

Once the specs are failing in expected ways, then modify the `edit`, `update`, and `destroy` actions to be more secure.